node-tests/blueprints/component-class-test.js

```diff
@@ -52,213 +52,134 @@ describe('Blueprint: component-class', function () {
-          ['component-class', '--component-structure', 'flat', 'foo'],
-          (_file) => {
-            expect(_file('app/components/foo.js')).to.equal(emberComponentContents);
-          }
-        );
-      });
-
-      it('component-class foo --component-structure=nested', function () {
-        return emberGenerateDestroy(
-          ['component-class', '--component-structure', 'nested', 'foo'],
-          (_file) => {
-            expect(_file('app/components/foo/index.js')).to.equal(emberComponentContents);
-          }
-        );
-      });
-
-      it('component-class foo --component-structure=classic', function () {
-        return emberGenerateDestroy(
-          ['component-class', '--component-structure', 'classic', 'foo'],
-          (_file) => {
-            expect(_file('app/components/foo.js')).to.equal(emberComponentContents);
-          }
-        );
-      });
-
-      it('component-class foo --component-class=@ember/component', function () {
-        return emberGenerateDestroy(
-          ['component-class', '--component-class', '@ember/component', 'foo'],
-          (_file) => {
-            expect(_file('app/components/foo.js')).to.equal(emberComponentContents);
-          }
-        );
-      });
-
-      it('component-class foo --component-class=@glimmer/component', function () {
-        return emberGenerateDestroy(
-          ['component-class', '--component-class', '@glimmer/component', 'foo'],
-          (_file) => {
-            expect(_file('app/components/foo.js')).to.equal(glimmerComponentContents);
-          }
-        );
-      });
-
-      it('component-class foo --component-class=@ember/component/template-only', function () {
-        return emberGenerateDestroy(
-          ['component-class', '--component-class', '@ember/component/template-only', 'foo'],
-          (_file) => {
-            expect(_file('app/components/foo.js')).to.equal(templateOnlyContents);
-          }
-        );
-      });
-    }
-
-    it('component-class x-foo', function () {
-      return emberGenerateDestroy(['component-class', 'x-foo'], (_file) => {
-        expect(_file('app/components/x-foo.js')).to.equal(fixture('component/component-dash.js'));
-      });
-    });
-
-    it('component-class foo/x-foo', function () {
-      return emberGenerateDestroy(['component-class', 'foo/x-foo'], (_file) => {
-        expect(_file('app/components/foo/x-foo.js')).to.equal(
-          fixture('component/component-nested.js')
-        );
+        expect(_file('app/components/foo.js')).to.equal(glimmerComponentContents);
       });
     });
 
-    it('component-class x-foo --path foo', function () {
-      return emberGenerateDestroy(['component-class', 'x-foo', '--path', 'foo'], (_file) => {
-        expect(_file('app/components/x-foo.js')).to.equal(fixture('component/component-dash.js'));
-      });
+    // classic default
+    it('component-class foo --component-structure=classic --component-class=@ember/component', function () {
```

Comment:
Can you double check what the RFC says RE: "--component-structure=classic"? I think we probably don't need that flag value anymore...

Reply:
For anyone referencing this later, after discussing with @rwjblue we decided this would be fine in a separate PR.